test-project/Assets/Scenes/SampleScene.unity

```diff
@@ -0,0 +1,258 @@
+%YAML 1.1
```

Comment:
`Tests.unity`?

Reply:
Sure 😄. The scene isn't used for anything yet 🔜 ™️